src/phongo_bson_encode.h

```diff
@@ -22,9 +22,10 @@
 #include <php.h>
 
 typedef enum {
-	PHONGO_BSON_NONE      = 0x00,
-	PHONGO_BSON_ADD_ID    = 0x01,
-	PHONGO_BSON_RETURN_ID = 0x02
+	PHONGO_BSON_NONE             = 0,
+	PHONGO_BSON_ADD_ID           = (1 << 0),
+	PHONGO_BSON_RETURN_ID        = (1 << 1),
+	PHONGO_BSON_ALLOW_ROOT_ARRAY = (1 << 2)
```

Comment:
The bit-shift notation here is consistent with various enum flags throughout libmongoc.

Reply:
Thanks for making the change. This is much easier to read for me.